src/main/java/com/google/devtools/build/lib/skyframe/TraversalRequest.java

```diff
@@ -52,6 +52,14 @@ public abstract class TraversalRequest implements SkyKey {
    */
   protected abstract boolean skipTestingForSubpackage();
 
+  /**
+   * Whether to emit nodes for empty directories.
+   *
+   * <p>If this returns false (the default), empty directories will not be represented in the result
```

Comment:
One more thing which came up from the internal review is this Javadoc. Would you be OK with dropping `(the default)` from it? There is no default implementation (sorry, I missed that).

Reply:
Oops didn't update it. Please do.